provider/server/machine_delete_test.go

```diff
@@ -17,8 +18,122 @@ import (
 // TODO: This test will require update after merge of PR: #101
 var _ = Describe("DeleteMachine", func() {
 
-	It("should delete a machine", func(ctx SpecContext) {
-		By("creating a machine")
+	It("should delete a machine with graceful shutdown", func(ctx SpecContext) {
+		By("creating a machine with ceph volume which boots properly ")
+		createResp, err := machineClient.CreateMachine(ctx, &iri.CreateMachineRequest{
+			Machine: &iri.Machine{
+				Metadata: &irimeta.ObjectMetadata{
+					Labels: map[string]string{
+						"foo": "bar",
+					},
+				},
+				Spec: &iri.MachineSpec{
+					Power: iri.Power_POWER_ON,
+					Image: &iri.ImageSpec{
```

Comment:
you have to use ceph volume with os. This option cannot be use in this case.

Reply:
I was just trying to get the vm booted with `ceph-volume` and OSimage for testing purpose, However we cannot do the ceph-volume boot at the moment because it would be required  to setup `ceph-provider` as well.
I will anyway proceed with ram boot for the integration test.

Reply:
do you have image which is bootable with image in spec?